### ISSUE TITLE
Pick from winning 

### ISSUE THREAD

**Issue by 14corman** [14corman/JMIProject#20]
Go over the best pre NN model and post NN model to see which one is best and why.

**Comment by 14corman** [comment]
By putting the 2 conclusions together from [best preNN](https://github.com/14corman/JMIProject/issues/7) and [best post NN](https://github.com/14corman/JMIProject/issues/18), here is what we get:
![image](https://user-images.githubusercontent.com/15676385/56130937-cf807900-5f4b-11e9-8050-a74494697172.png)

![image](https://user-images.githubusercontent.com/15676385/56130960-dd35fe80-5f4b-11e9-9c55-13a98e6489b4.png)

![image](https://user-images.githubusercontent.com/15676385/56130983-eaeb8400-5f4b-11e9-9e73-1e9a5a73dd0a.png)

The pink and light blue are the 2 lines we are interested in as the pink is the post winning model and light blue is the pre winning model. As you can see, they both use 0.2 drop out and 0.001 learning rate which is interesting. The pink is only slightly better than the blue by the tiniest of margins. It was also able to keep the training more stable than the light blue. I am assuming both of these are because of the variables' correlation, but the fact that the improvement was not that much better means that the correlation was not good enough. 

I think the winning model should be the post model due to all of that. That means the 2 extra variables do help, but you have to be careful because the network can start overfitting very easily even with bath normalization and drop out.

**Comment by 14corman** [comment]
The test accuracy for the winning model is 67%. The other model got 69% test accuracy (one without demographic info). I think the extra variables are having the model overfit too much. I think we should stick with the older winning model without the demographic variables because of this.

**Comment by 14corman** [comment]
I just calculated their test accuracies and test f1 scores:

pre-model test:
    accuracy: 0.6650558758737662
    f1 score: 0.4715201089089613

post-model test:
    accuracy: 0.6650383973288815
    f1 score: 0.4714964477565678

Its weird that both models actually perform exactly the same nearly... However, I guess that means in the end those extra variables do nothing.
